package.nls.json

```diff
@@ -171,6 +171,7 @@
     "AWS.command.viewSchemaItem": "View Schema",
     "AWS.command.searchSchema": "Search Schemas",
     "AWS.command.executeStateMachine": "Start Execution...",
+    "AWS.command.renderStateMachineGraph": "Render graph",
```

Comment:
Why do we call this "Render graph"? That seems a bit technical. Is "Visualize" more intuitive / conventional?

Reply:
We already use "Render" in CodeLenses and Command Palette so I'm keeping it consistent here. Also, that is an option preferred by our UX team.

Reply:
Definitely in favor of consistency.